src/utils.py

```diff
@@ -274,7 +274,7 @@ def pr_build_setup(pr_number, framework):
 
     # The below code currently overides the device_types, image_types, py_versions with constants.ALL
     # when there is a change in any the below files
-    parse_modifed_buidspec_yml_info(files, framework, pattern="\S+\/buildspec.yml")
+    parse_modifed_buidspec_yml_info(files, framework, pattern="\S+\/buildspec.*yml")
```

Comment:
`buildspec*.yml` ?

Reply:
It is change in regex to allow buildspec-tf1 or anything similiar to that as pattern
.* to have zero or more character

Reply:
Otherwise the tf-1 builds does not start automatically as in the earlier iteration in morning. @Satish615 suggested this change.

Reply:
You might have a typo here, as Nikhil mentioned

Reply:
I think what nikhil points out is  buildspec*.yml will only consider yml files starting with buildspec but if you use  buildspec.*yml then it will consider any files which has buildspec and yml inside it. 
its unlikely that we will have yml inside the filename but lets not assume that.

Reply:
@YYStreet That is not a typo, I have changed the pattern for regex to include buildspec file with name like buildspec-tf1